package.json

```diff
@@ -32,6 +32,7 @@
     "bundle": "node ./scripts/docs/bundle.js",
     "lint": "jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/",
     "test": "npm run docs && mocha --timeout 5000 --bail test/docs.js packages/*/test/*.js",
```

Comment:
Let's just change this to `npm run docs && npm run snippet-docs && npm run unit-test`

"unit-test": "mocha --timeout 5000 --bail packages/*/test/*.js"